tests/test_md_moves.py

```diff
@@ -0,0 +1,38 @@
+from typing import Tuple
+
+import numpy as np
+from scipy.stats import ks_2samp
+
+from timemachine.md.moves import MetropolisHastingsMove
+
+
+def test_rwmh():
```

Comment:
nit: For clarity when running tests
```suggestion
def test_random_walk_metropolis_hastings():
```